python/src/SolverImplementation_doc.i.in

```diff
@@ -167,3 +167,31 @@ nEval : int
 %enddef
 %feature("docstring") OT::SolverImplementation::getCallsNumber
 OT_Solver_getCallsNumber_doc
+
+// ---------------------------------------------------------------------
+
+%define OT_Solver_solve_doc
+"Solve an equation.
+
+Parameters
+----------
+function : :class:`~openturns.Function`
+    The function of the equation :math:`function(x) = value` to be solved in the
+    interval :math:`[infPoint, supPoint]`.
+value : float
+    The value to which the function must be equal.
+infPoint, supPoint : float
+    Lower and upper bounds of the interval definition of the variable :math:`x`.
```

Comment:
What does "the interval definition of x" mean?

Reply:
yes, lets use "range" instead